src/repository/repository.reducer.js

```diff
@@ -62,19 +63,21 @@ export const repositoryReducer = (state = initialState, action = {}) => {
       return {
         ...state,
         issues: [],
+        error: '',
```

Comment:
If you persist to keep 2 fields, I suggest to clear `hasRepoExist` when pending and error. If repo1 set it to be `true`, then repo2 will get wrong value during pending or error.

Reply:
@chinesedfan in current changes, i've tried to test in real world. I don't face any problem when openning from one repo to another.

Reply:
Yes, as you reset with `initialState`, the error state is cleared. And the pending state is too fast to be noticed.

Reply:
Understand, okay i will set `hasRepoExist` when pending too